packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -1,31 +1,25 @@
 import * as React from 'react';
 import { View } from 'react-native';
-import { ShadowProps } from './Shadow.types';
+import { ShadowProps, ShadowTokens, shadowName } from './Shadow.types';
+import { compressible, buildUseTokens, useFluentTheme } from '@fluentui-react-native/framework';
+import { shadowStyleFromTheme } from './shadowStyle';
 
-export const Shadow: React.FunctionComponent<ShadowProps> = (props: ShadowProps) => {
-  const noOffset = { width: 0, height: 0 };
-  const neutralShadowAmbient = { shadowOffset: noOffset, shadowOpacity: 0.12, shadowRadius: 2 };
-  const neutralShadowAmbientDarker = { shadowOffset: noOffset, shadowOpacity: 0.2, shadowRadius: 8 };
-  const ambientShadow = {
-    '2': neutralShadowAmbient,
-    '4': neutralShadowAmbient,
-    '8': neutralShadowAmbient,
-    '16': neutralShadowAmbient,
-    '28': neutralShadowAmbientDarker,
-    '64': neutralShadowAmbientDarker,
-  };
-  const keyShadow = {
-    '2': { shadowOffset: { width: 0, height: 1 }, shadowOpacity: 0.14, shadowRadius: 2 },
-    '4': { shadowOffset: { width: 0, height: 2 }, shadowOpacity: 0.14, shadowRadius: 4 },
-    '8': { shadowOffset: { width: 0, height: 4 }, shadowOpacity: 0.14, shadowRadius: 8 },
-    '16': { shadowOffset: { width: 0, height: 8 }, shadowOpacity: 0.14, shadowRadius: 16 },
-    '28': { shadowOffset: { width: 0, height: 14 }, shadowOpacity: 0.24, shadowRadius: 28 },
-    '64': { shadowOffset: { width: 0, height: 32 }, shadowOpacity: 0.24, shadowRadius: 64 },
+const useShadowTokens = buildUseTokens<ShadowTokens>(() => ({}), shadowName);
+
+export const Shadow = compressible<ShadowProps, ShadowTokens>((props: ShadowProps) => {
+  const { depth } = props;
+  const theme = useFluentTheme();
+
+  const shadowStyle = shadowStyleFromTheme(theme, 'shadow' + depth.toString());
```

Comment:
Not sure about the 'shadow' + depth.toString() part, thinking I can either make ShadowDepth to be "shadow2, shadow4, ..." instead of "2, 4, ..."

Reply:
Updated to take in the type ShadowDepth, but ended up also making ShadowDepth "shadow2, shadow4, etc" because I realized that there are also different shadows for brands (ex. shadow2brand). 

@rurikoaraki I noticed that [in the PR you made adding shadow tokens](https://github.com/rurikoaraki/fluentui-design-tokens/blob/e0fb4335a666ed62c08812bb87d1bc33826b2bad/src/shadow.json), the object is divided into 'global' and 'brand' shadows, and then within each group there's 2, 4, 6, 8, etc.  but in FURN at some point the depths and global vs. brand get flattened into BaseShadowAliasTokens (i.e. shadow2, shadow2brand, etc). Just wanted to check with you that having the ShadowDepth type match BaseShadowAliasTokens makes the most sense?